src/installer/pkg/projects/Microsoft.NETCore.DotNetHost/Microsoft.NETCore.DotNetHost.pkgproj

```diff
@@ -27,7 +27,7 @@
     <Copy SourceFiles="$(DotNetHostBinDir)/dotnet$(ApplicationFileExtension)"
           DestinationFolder="$(PublishRootDir)" />
 
-    <Copy SourceFiles="$(RepoRoot)THIRD-PARTY-NOTICES.TXT"
+    <Copy SourceFiles="$(InstallerProjectRoot)pkg\THIRD-PARTY-NOTICES.TXT"
```

Comment:
Are the copies in the root and under the pkg folder the same (given that we used to pick up from the root copy)? If so, why do we have duplicate files? @wli3 any idea?

Reply:
I'm introducing the second file, to be shipped and installed to dotnet folder. The file in repo root should only cover TPNs for that repo (runtime in this case).

New file is not a simple copy, it has TPNs from runtime repo and all others that install content under dotnet folder.